batch/batch/server/server.py

```diff
@@ -785,18 +799,22 @@ def from_record(record):
                                        user=user)
 
         batch = Batch(id=id, attributes=attributes, callback=callback,
-                      ttl=ttl, is_open=is_open, userdata=userdata, user=user)
+                      ttl=ttl, is_open=is_open, userdata=userdata, user=user,
+                      state='running', complete=False)
         batch.close_ttl()
         return batch
 
-    def __init__(self, id, attributes, callback, ttl, is_open, userdata, user):
+    def __init__(self, id, attributes, callback, ttl, is_open, userdata, user,
+                 state, complete):
         self.id = id
         self.attributes = attributes
         self.callback = callback
         self.ttl = ttl
         self.is_open = is_open
         self.userdata = userdata
         self.user = user
+        self.state = state
+        self.complete = complete
 
     async def get_jobs(self):
         job_ids = await db.batch_jobs.get_jobs(self.id)
```

Comment:
This still needs to get fixed.  `get_jobs` should just query and return the full records.

Reply:
Oops, this was in another PR, sorry.